pkg/demoinfocs/common/hostage.go

```diff
@@ -0,0 +1,65 @@
+package common
+
+import (
+	"github.com/golang/geo/r3"
+
+	st "github.com/markus-wa/demoinfocs-golang/v2/pkg/demoinfocs/sendtables"
+)
+
+// HostageState is the type for the various HostageStateXYZ constants.
+type HostageState byte
+
+// HostageState constants give information about hostages state.
+// e.g. being untied, picked up, rescued etc.
+const (
+	HostageStateIdle            HostageState = 0
+	HostageStateBeingUntied     HostageState = 1
+	HostageStateGettingPickedUp HostageState = 2
+	HostageStateBeingCarried    HostageState = 3
+	HostageStateFollowingPlayer HostageState = 4
+	HostageStateGettingDropped  HostageState = 5
+	HostageStateRescued         HostageState = 6
+	HostageStateDead            HostageState = 7
+)
+
+// Hostage represents a hostage.
+type Hostage struct {
+	Entity           st.Entity
+	demoInfoProvider demoInfoProvider
+}
+
+// Position returns the current position of the hostage.
+func (hostage *Hostage) Position() r3.Vector {
+	if hostage.Entity == nil {
+		return r3.Vector{}
+	}
+
+	return hostage.Entity.Position()
```

Comment:
Is this working as expected?
I'm worried this might be causing problems: https://github.com/markus-wa/demoinfocs-golang/blob/master/pkg/demoinfocs/sendtables/entity.go#L207-L242

I'll have a more detailed look later to confirm

Reply:
I did a quick test by standing on the head of a hostage 😄 and it looks fine:

```
akiver (1721.302978515625000000000000, 753.940368652343750000000000, -95.837493896484375000000000)
Hostage (1720.000000000000000000000000, 752.000000000000000000000000, -159.875000000000000000000000)
```

Reply:
awesome, thanks for the quick check!